setup.py

```diff
@@ -13,7 +13,7 @@
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-VERSION = "5.3.0"
+VERSION = "5.3.1"
```

Comment:
I'm assuming we want to bump the version like this.

Reply:
Yes 👍🏻